3DAmsterdam/Assets/Amsterdam3D/Scripts/Traffic Animation/RoadPoint.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RoadPoint : MonoBehaviour
+{
+    public Vector3 pointCoordinates;
+    public void Initiate(double longitude, double latitude)
+    {
+        Vector3 tempCoordinates = ConvertCoordinates.CoordConvert.WGS84toUnity(longitude, latitude);
+        tempCoordinates.y = 45f; // random height to make sure its somewhat close to the map.
```

Comment:
Hier zou je ook Constants.ZERO_GROUND_LEVEL_Y voor kunnen gebruiken (die zit op 43, en dat is het Unity y coordinaat wat ongeveer ter hoogte van het maaiveld zit. Dan blijft dat mooi op 1 plek.
Of als het bewust 2m hoger is kan je Constants.ZERO_GROUND_LEVEL_Y + 2.0f; doen